pkg/commands/workload.go

```diff
@@ -124,6 +125,7 @@ func (opts *WorkloadOptions) Validate(ctx context.Context) validation.FieldError
 	errs = errs.Also(validation.DeletableKeyValues(opts.Labels, flags.LabelFlagName))
 	errs = errs.Also(validation.DeletableKeyValues(opts.Annotations, flags.AnnotationFlagName))
 	errs = errs.Also(validation.DeletableKeyValues(opts.Params, flags.ParamFlagName))
+	errs = errs.Also(validation.DeletableKeyValues(opts.ParamsYaml, flags.ParamYamlFlagName))
```

Comment:
*issue* invalid yaml should to produce an error here. This line also needs test coverage

Reply:
The yaml/json validation was moved